src/modules/OTA-api/data-components/Contributors.tsx

```diff
@@ -14,19 +16,43 @@ type ContributorsProps = {
   marginTop?: boolean;
 } & React.HTMLAttributes<HTMLDivElement>;
 
+const mapAllContributorsData = (
+  allContributors: {
+    name: string;
+    profile: string;
+    avatar_url: string;
+  }[]
+) => {
+  const contributorsShemed: {
+    [name: string]: {
+      website: string;
+      picture?: string;
+    };
+  } = {};
+  Object.values(allContributors).map((contributor: any) => {
+    contributorsShemed[contributor.name] = {
+      website: contributor.profile,
+    };
+    if (contributor.avatar_url != '') {
+      contributorsShemed[contributor.name].picture = contributor.avatar_url;
+    }
+  });
+  return contributorsShemed;
+};
+
 const getContributorsByType = (type: ContributorsProps['type']) => {
   switch (type) {
     case 'core':
-      return contributorsData.core;
+      return staffData.core;
     case 'alumnis':
-      return contributorsData.alumnis;
-    case 'contributors':
-      return contributorsData.contributors;
+      return staffData.alumnis;
     case 'communityManagers':
-      return contributorsData.communityManagers;
+      return staffData.communityManagers;
+    case 'contributors':
+      return mapAllContributorsData(allContributorsData.contributors);
     case 'all':
     default:
-      return contributorsData;
+      return staffData;
```

Comment:
Unless `all` is effectively used, wouldn't this suffice? 🙂 

> We could even simplify further by unifying the formats and loading the data into the same object, removing the need for a function altogether.

```suggestion
if (type == 'contributors')
  return mapAllContributorsData(allContributorsData.contributors);

return staffData[type];
```

Reply:
It's entirely possible to improve data management by having a single unified object and deleting what isn't used, but I've chosen to go as quickly as possible. 

`core` `alumnis` and `communityManagers` are used.